packages/governance/src/closure.js

```diff
@@ -0,0 +1,19 @@
+// @ts-check
+
+// Standard closure rule is a deadline and Timer. An alternative might allow for
```

Comment:
"Closure" is confusing to me both because of it's comp sci use, and because it usually refers to final completion rather than the process of closing.

Reply:
changed to closingRule.